crates/conformance-test-utils/src/utils.rs

```diff
@@ -0,0 +1,58 @@
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_contract_address;
+
+/// Computes the StarkNet address of EVM address given the Kakarot address and Account Proxy Class
+/// Hash. Uses the same logic as get_contract_address with EVM as salt.
+pub fn compute_starknet_address(
+    kakarot_address: FieldElement,
+    account_proxy_class_hash: FieldElement,
+    evm_address: FieldElement,
+) -> FieldElement {
+    let constructor_calldata: Vec<FieldElement> = Vec::new();
+    get_contract_address(evm_address, account_proxy_class_hash, &constructor_calldata, kakarot_address)
+}
+
+#[cfg(test)]
+mod tests {
+    use starknet::core::crypto::compute_hash_on_elements;
+    use starknet::core::types::FieldElement;
+    use starknet::core::utils::normalize_address;
+
+    use super::compute_starknet_address;
+
+    #[test]
+    fn test_compute_starknet_address() {
+        // From https://github.com/xJonathanLEI/starknet-rs/blob/283c2bb814a964a49d67aafadc4340c5fb714fdd/starknet-core/src/utils.rs#L12
```

Comment:
Actual references for the computing is https://github.com/starkware-libs/cairo-lang/blob/master/src/starkware/starknet/core/os/contract_address/contract_address.cairo

Reply:
Thanks I'll update this